src/Sylius/Bundle/ApiBundle/CommandHandler/Account/VerifyCustomerAccountHandler.php

```diff
@@ -34,7 +33,7 @@ public function __construct(
     ) {
     }
 
-    public function __invoke(VerifyCustomerAccount $command): JsonResponse
+    public function __invoke(VerifyCustomerAccount $command): void
```

Comment:
Below is a line:
`sprintf('There is no shop user with %s`
It would be good to change it to:
`sprintf('There is no shop user with "%s"`